staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go

```diff
@@ -2236,18 +2237,20 @@ func (t *fancyTransformer) createObject() error {
 		},
 	}
 	out := &example.Pod{}
-	return t.store.Create(context.TODO(), key, obj, out, 0)
+	return t.store.Create(t.ctx, key, obj, out, 0)
```

Comment:
this should be plumbed as a parameter to createObject (and should use the parameter already passed to TransformFromStorage, for example)

Reply:
from context godoc:

> Do not store Contexts inside a struct type; instead, pass a Context explicitly to each function that needs it. The Context should be the first parameter, typically named ctx

Reply:
@liggitt I considered trying to do that, but I'm not sure it's possible - this is called as a side-effect of transforming to storage, and there is no request context present in that call. The `context.Context` there is used for figuring out encryption.

Reply:
oh, that's funny... different Context interface

it's not clear why a background context in the test struct is more correct than a TODO context passed directly to the storage create call

Reply:
It's "more correct" in that one, coherent context is used in all calls to the `storage.Interface` during the whole test. In this file there's quite a lot of copy-pasta in general, but this looks to be one of the few tests that did not get changed to use `testSetup()` which divulges (a background) context that tests use to pass to all calls. For no reason other than making this not the odd-one-out, I think it's worth it.